packages/vite/src/node/constants.ts

```diff
@@ -47,7 +47,7 @@ export const DEFAULT_CONFIG_FILES = [
 export const JS_TYPES_RE = /\.(?:j|t)sx?$|\.mjs$/
 
 export const CSS_LANGS_RE =
-  /\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)($|\\?)/
+  /\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)(?:$|\?)/
```

Comment:
I checked the codebase and found that we don't use the second capture group, so added a `?:` so it's stricter for now.